### DISCUSSION TITLE
Do we really need at lease two servers, one running wordpress, one running woonuxt to make this to work? Thank you.

### DISCUSSION THREAD

**Discussion by mobilelifeful** [scottyzen/woonuxt#243] [Q&A]
When I installed wordpress and woonuxt one the same server, no matter what I tried, not working for me, but once I put woonuxt on different server, it seems start working, although it still need to figure out how to add ssl to the woonuxt.

**Comment by fabdao** [comment]
No you don´t. You only need one server. The only thing witch will differ of traditional WordPress architecture will be the stacks. You will need on your server apache or nginx for serve php/html files plus nodejs / npm for serve headless vuejs part. 
But all will run on the same machine. 

Here all the stack on raspberry pie running fine : https://leading-exactly-lobster.ngrok-free.app

The shop is disable on purpose but it's working great.
